README.md

```diff
@@ -30,3 +30,10 @@ Temperature and Magnet controllers:
 
 ## Live plotting using PyQtGraph
 Measurement results are plotted live using PyQtGraph.
+
+## Future work
+- Deploy to Janis, 9T, 12T and 16 system
+- Modular design for TDaemon and MDaemon to allow more robust support of vairous systems.
+- Support vector magnet
+- Variable step size in do_device_sweep and do_fridge_sweep
+- Logging to file in TDaemon and MDaemon
```

Comment:
You can add more new features into future work. Please have a look at issues #9 #10 .